KubeArmor/enforcer/runtimeEnforcer.go

```diff
@@ -164,7 +202,16 @@ func (re *RuntimeEnforcer) DestroyRuntimeEnforcer() error {
 
 	errorLSM := false
 
-	if re.EnforcerType == "AppArmor" {
+	if re.EnforcerType == "BPFLSM" {
+		if re.bpfEnforcer != nil {
+			if err := re.bpfEnforcer.DestroyBPFEnforcer(); err != nil {
```

Comment:
Should we explicitly set the bpfEnforcer to nil after destroying it? If yes, same applies to Apparmor and SELinux below.